apollos-church-api/src/data/index.js

```diff
@@ -73,9 +88,12 @@ const data = {
   Event,
   Cache,
   PrayerRequest,
+  Comment,
+  UserFlag,
+  Follow,
+  PostgresCampus,
   Persona,
-  Person,
-  BinaryFiles,
+  NoPostgres: ApollosConfig.DATABASE_URL ? NoPostgres : {}, // this needs to be last to overwrite all Postgres resolvers
```

Comment:
I like this approach. How do you feel about keeping two files included for now though (index.postgres.js and index.js) that will make resolving conflicts easier when we run the update tool, I think.

Reply:
my opinion is we bite the bullet and the next upgrades are potentially more difficult in order to get rid of unnecessary confusing logic and files

Reply:
I would agree with you, but hard upgrades seem to === more mistakes and more bugs :/ I like maintaining two distinct paths more than one main path with switchbacks. Maybe there's a compromise solution though? A way we could isolate the Postgres files better in one file?

Reply:
We _could_ throw the `NoPostgres` pkg in the old `index.js` file only (leave it off `index.postgres.js`) but that means every time we do a PR we have to consider if new pkgs go in one or the other or both index files. This way, we can just include it like it will be for 3.0 and add the mocks if necessary. And when postgres is required for all, we just pull the mocks file out and it just works. You think the upgrades will be that difficult? Or do you like the two separate file regardless of upgrade difficulty?

Reply:
I like the idea of throwing the `NoPostgres` pkg in the old `index.js` file only. It's pretty infrequent we add new modules, and when we do we will likely add them _only_ to the `index.postgres.js` file and the `NoPostgres` file (if they add new schema.

A few more reasons that make a single file approach hard - this file  right now won't work because the People dataSource comes from `rockWithPostgres`. It needs to come from Rock, and the `rockWIthPostgres` modules need to be ignored, but only when not using . Same thing with OneSignal.